vale/code/crypto/aes/ppc64le/Main.ml

```diff
@@ -0,0 +1,26 @@
+let parse_cmdline :
```

Comment:
Could you put this with the cmdline parsing for x64 in code/lib/util/CmdLineParser.ml, and rename both functions in, say, parse_cmdline_x64 and parse_cmdline_ppc64le?

Reply:
Done!